qutebrowser/misc/notree.py

```diff
@@ -0,0 +1,240 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
This is something I was thinking about. I didn't do it yet because I almost never used type hints before, so I'm not comfortable using them without learning more first. 

I'll read around and eventually add annotations.

Reply:
If you would like, I'd be fine with adding type annotations to this file for you (and push it to your branch).

Reply:
Wonderful job, thank you!